workshops/templates/workshops/event.html

```diff
@@ -6,6 +6,17 @@
 {% load tags %}
 
 {% block content %}
+
+{% if event.tags_changed %}
```

Comment:
Yes, if there are any changes detected (e.g. event website was updated and new instructors were set, or new contact details were specified) the `event.tags_changed` will be set to `True`, and the notification in this template will show up.